index.html

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Learning HTML and CSS</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+  	<div class = "title">
+  		<h1>Git</h1>
+		<p>This webpage will answer these 3 questions.</p>
+		<p>What is Git?</p>
+		<p>Why do developers need Git?</p>
+		<p>What is a branch in Git?</p>
+	</div>
+		
+	<main>
+		<div class="parent">
+		    <div> <img src = "git.png" alt = "git logo" class = "center">
+	    	<article>
+			<h3>What is Git?</h3>
+				<p>Git basically takes a picture of what all your files look like at that moment and stores a reference to that snapshot.</p>
+			<nav><a href = "https://git-scm.com/book/en/v2/Getting-Started-What-is-Git%3F" target = "_blank">Read more</a></nav>
+			</article>
+			</div>    
+		</div>
+	</main>
+
+<section>
+	<div class="parent1">
+    	<div>
+    		<img src = "developers.png" alt = "git logo"  class  = "img1">
+			<article>
+			<h3>Why do developers need Git?</h3>
+			<p>Git is the most commonly used version control system. Git tracks the changes you make to files, so you have a record of what has been done, and you can revert to specific versions should you ever need to. Git also makes collaboration easier, allowing changes by multiple people to all be merged into one source.</p>
+			<nav><a href = "https://git-scm.com/about/branching-and-merging" target = "_blank">Read more</a></nav>
+			</article>
+		</div>
+    	
+    	<div>
+    		<img src = "gitbranch.png" alt = "git logo" width = "100%">
```

Comment:
Is the image part of the article? If so, should it be grouped inside the article?

Reply:
Hi 

I have made the changes

Reply:
Yes and I have grouped it inside the article element in my updated index.html